msal_extensions/cache_lock.py

```diff
@@ -25,7 +28,24 @@ def __init__(self, lockfile_path):
             flags=portalocker.LOCK_EX | portalocker.LOCK_NB,
             **open_kwargs)
 
+    def try_to_create_lock_file(self):
+        retries_no = 100
+        retry_delay_milliseconds = 0.150
+        for i in range(retries_no):
+            try:
+                with open(self._lockpath, 'x'):
+                    return True
+            except OSError as err:
+                if err.errno == errno.EEXIST:
+                    time.sleep(retry_delay_milliseconds)
```

Comment:
[Isn't this](https://docs.python.org/3/library/time.html#time.sleep) `retry_delay_in_secs`, rather than `milliseconds`?

Reply:
Good catch, will make the change!